code/modules/psionics/mob/mob_assay.dm

```diff
@@ -50,6 +50,9 @@
 				if(5)
 					use_rating = "<font color = '#FF0000'>[effective_rating]-Alpha</font>"
 					rating_descriptor = "This indicates the presence of major psi capabilities of the Paramount Grandmaster rank or higher."
```

Comment:
Since you now have the Infinity class, you can remove the `[...] or higher. [...]` since there's an actual classification for it now.

Reply:
Fair enough.